hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncTableRegionLocatorImpl.java

```diff
@@ -47,19 +49,21 @@ public TableName getName() {
 
   @Override
   public CompletableFuture<HRegionLocation> getRegionLocation(byte[] row, int replicaId,
-      boolean reload) {
+    boolean reload) {
     return conn.getLocator().getRegionLocation(tableName, row, replicaId, RegionLocateType.CURRENT,
       reload, -1L);
   }
 
   @Override
   public CompletableFuture<List<HRegionLocation>> getAllRegionLocations() {
-    if (TableName.isMetaTableName(tableName)) {
-      return conn.registry.getMetaRegionLocations()
-        .thenApply(locs -> Arrays.asList(locs.getRegionLocations()));
-    }
-    return ClientMetaTableAccessor
-      .getTableHRegionLocations(conn.getTable(TableName.META_TABLE_NAME), tableName);
+    return tracedFuture(() -> {
+      if (TableName.isMetaTableName(tableName)) {
+        return conn.registry.getMetaRegionLocations()
+          .thenApply(locs -> Arrays.asList(locs.getRegionLocations()));
+      }
+      return ClientMetaTableAccessor
+        .getTableHRegionLocations(conn.getTable(TableName.META_TABLE_NAME), tableName);
+    }, getClass().getSimpleName() + ".getAllRegionLocations");
```

Comment:
Above you just do getName but here getSimpleName. What you thinking? Is full classname needed sometimes and other times not?

Reply:
Or annotation w/ servername doesn't make sense here?

Reply:
Should all be getSimpleName. Let me check.